src/utilities/new_versions.jl

```diff
@@ -216,6 +216,7 @@ function make_pr_for_new_version(
     subdir::String,
     local_clone_path::AbstractString,
     dep_section::String,
+    remote_name::AbstractString="compathelper-remote",
```

Comment:
Does this need to be a configurable parameter? I think it's fine for us to just set `CONST COMPATHELPER_REMOTE = "compathelper-remote"` and then use that.